adserver/templates/adserver/advertiser/flight-list.html

```diff
@@ -31,7 +31,7 @@ <h1>{% block heading %}{% trans "Manage advertising flights" %}{% endblock headi
   {% if has_advertiser_edit_permission %}
   <a href="{% url 'flight_request' advertiser.slug %}" class="btn btn-sm btn-outline-info mb-3" role="button" aria-pressed="true">
     <span class="fa fa-plus mr-1" aria-hidden="true"></span>
-    <span>{% trans 'Request a new flight' %}</span>
+    <span>{% trans 'Renew or request a new flight' %}</span>
```

Comment:
This feels a little hacky to note both here -- I don't know if a single bit of feedback is worth updating all the UI items? 

Do we want something like `Set up a new flight`, since that's what is actually happening?